src/components/SearchMetadata.vue

```diff
@@ -11,23 +13,17 @@ export default {
   name: "SearchMetadata",
   props: {
     hits: Number,
-    query: String,
-    status: Object,
+    searchterm: String,
   },
-  computed: {
-    panelHeading: function () {
-      if (this.status.ready == true) {
-        return "Results summary: " + this.hits + " results";
-      } else if (this.status.loading == true) {
-        return "Loading results...";
-      }
-      return "Search Metadata:";
+  methods: {
+    per_page() {
+      return 20;
     },
-    panelStatus: function () {
-      if (this.status.ready == true) {
-        return "panel-success";
-      }
-      return "panel-info";
+    start() {
+      return this.$route.query.page * this.per_page() - this.per_page() + 1;
```

Comment:
This is mathematically correct, and I don't think we _need_ to make it more efficient - but it can be done without calling per_page() twice if the expression is `this.per_page() * ( this.$route.query.page - 1 ) + 1`

Reply:
#team_no_math. But also I'll look closer at that expression and see if I feel like it makes it clearer or less clear what we're doing. 👍🏻

Reply:
Yeah I like this better. I'm going to make this change. Thanks for the suggestion.